WordPress/src/main/res/values/strings.xml

```diff
@@ -2698,6 +2698,18 @@
     <string name="domains_free_plan_get_your_domain_caption">Domains purchased on this site will redirect visitors to &lt;b&gt;%s&lt;/b&gt;</string>
     <string name="domains_search_for_a_domain_button">Search for a domain</string>
 
+    <!-- Domain Management -->
+    <string name="domain_management_status_complete_setup">Complete Setup</string>
```

Comment:
My understanding is that those are not used at this point and it is pending to decide if we will use the [localised server status string](https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/trunk/fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpcom/site/AllDomainsResponse.kt#L53) instead. Is that correct?

Reply:
That's correct, afaiu. The only exception is the error case where a status is somehow `null`. In that case, we can use an app-translated "Error" string to convey some error state to the user. Resolved in https://github.com/wordpress-mobile/WordPress-Android/pull/19269/commits/92930619e31ac0c64f05390ee98289bd29b06bae.

Reply:
Thank you for the change @mkevins 🙇